libs/cypress-indexeddb/src/lib/object-store-CRUD.ts

```diff
@@ -68,6 +102,18 @@ export function createItem(
   return setItem('add', store, key, value);
 }
 
+/**
+ * Update item for the provided store key
+ *
+ * @remarks The `updateItem` method updates the value for the key provided. You can chain assertions from this method. If you use TypeScript, you can set the type of the returned value
+ *
+ * @param store `IDBObjectStore` instance
+ * @param key item key
+ * @param value item value
+ *
+ * @returns IDBObjectStore
+ * @throws {Error} If the connections fails to open or the read operation fails.
```

Comment:
technically it only throws an error if it is chained off from a method that does not return an object store.